src/meta/meta_bulk_load_service.h

```diff
@@ -440,6 +459,8 @@ class bulk_load_service
     std::unordered_map<gpid, bool> _partitions_cleaned_up;
     // Used for bulk load failed and app unavailable to avoid duplicated clean up
     std::unordered_map<app_id, bool> _apps_cleaning_up;
+    // indicate that have the app ever get into ingesting status
+    std::unordered_map<app_id, bool> _apps_ever_ingesting;
```

Comment:
I think `_apps_ever_ingesting` should be store on zk, not in meta server memory. If an app ever get into ingesting, and meta server crashed, this value will be lost, then it may get wrong bulk_load_result. I suggest adding this filed into `app_bulk_load_info`, which will store onto zk during bulk load process.